test/DomainRegistry.js

```diff
@@ -0,0 +1,227 @@
+const { ethers } = require("hardhat");
+const { expect } = require("chai");
+const { loadFixture } = require("@nomicfoundation/hardhat-network-helpers");
+
+
+describe("DomainRegistry", function () {
+    async function deployDomainRegistryFixture() {
+        const PWEI_DECIMAL_PLACES_NUMBER = 15;
+        const registrationFee = ethers.parseUnits("15", PWEI_DECIMAL_PLACES_NUMBER); // 15 Finney
+
+        const [owner, addr1, addr2] = await ethers.getSigners();
+        const DomainRegistry = await ethers.getContractFactory("DomainRegistry");
+
+        const domainRegistry = await DomainRegistry.deploy(registrationFee);
+        await domainRegistry.waitForDeployment();
+
+        return { domainRegistry, registrationFee, owner, addr1, addr2 };
+    }
+
+    describe("Deployment", function () {
+        it("Should set the right owner and registration fee", async function () {
+            const { domainRegistry, registrationFee, owner } = await loadFixture(deployDomainRegistryFixture);
```

Comment:
line 22 and same places
if this line is moved to Before section, it will be more clear
// - 1 point